src/client/components/pages/entities/author.js

```diff
@@ -81,6 +81,11 @@ function AuthorAttributes({author, averageRating}) {
 								stars={5}
 							/>
 						</dd>
+						{reviewsCount ?
+							<dd className="small text-muted">
+								{reviewsCount} Review
```

Comment:
We can add a conditional 's' when there are multiple reviews, and using the same capitalization as in the cb-review component:
```suggestion
								{reviewsCount} review{reviewsCount > 1 ? 's' : ''}
```

Since the same code is repeated for each entity, I'm thinking we should have a separate review component where we can isolate this logic, and reuse that component for all entities.

Reply:
Done